src/internal/middleware.js

```diff
@@ -37,20 +37,24 @@ export default function sagaMiddlewareFactory(options = {}) {
   function sagaMiddleware({getState, dispatch}) {
     runSagaDynamically = runSaga
     const sagaEmitter = emitter()
+    const sagaDispatch = !sagaMonitor ? dispatch : wrapSagaDispatch(dispatch)
 
-    function runSaga(saga, ...args) {
+    function runSaga(saga, args, sagaId) {
```

Comment:
No that's an internal function, the public one is declared on the bottom `sagaMiddleware.run = (saga, ...args) = ...`